vuepress/docs/next/tutorials/solution/entando-hub.md

```diff
@@ -5,24 +5,30 @@ sidebarDepth: 2
 # Entando Hub
 ## Overview
 
-The Entando Hub enables a team to share components across Entando Applications. It can be installed in Entando 6.3.2 or higher and includes API-level integration with the Entando 7.0 App Builder.
+The enterprise Entando Hub enables teams to share components across the organization and between Entando Applications. It can be installed in Entando 6.3.2 or higher and includes API-level integration with the Entando 7.0 App Builder.
```

Comment:
This sentence looks like it's only describing the enterprise Hub which is the third option below. I think the original sentence was better for a description of the Hub in general. Just my 2 cents.